src/styles/blocks/catalog.scss

```diff
@@ -11,6 +13,7 @@
     justify-content: center;
     align-items: center;
     box-sizing: border-box;
+    transition: $transition-time;
```

Comment:
specify css property for transition
by default  `all` , but it is not good for performance